fieldservice/models/res_config_settings.py

```diff
@@ -108,4 +108,4 @@ def _onchange_module_fieldservice_stock(self):
     @api.onchange("module_fieldservice_purchase")
     def _onchange_module_fieldservice_purchase(self):
         if self.module_fieldservice_purchase:
```

Comment:
```suggestion

```
We should be able to remove this completely since the group has been completely removed from Odoo